corehq/apps/users/views/web.py

```diff
@@ -66,10 +66,16 @@ def __call__(self, request, uuid, **kwargs):
                                       "request a project administrator to send you the invitation again."))
             return HttpResponseRedirect(reverse("login"))
 
+        is_invited_user = False
+        if request.user.is_authenticated:
+            if request.couch_user.username.lower() == invitation.email.lower():
+                is_invited_user = True
```

Comment:
nit: what would you think about condensing this a bit (one possibility below)? Though I do see the nested `if`s used throughout this view so I can also see it might be clearer to follow that pattern.
```suggestion
        is_invited_user = (request.user.is_authenticated
            and request.couch_user.username.lower() == invitation.email.lower())
```

Reply:
Yes, that's nicer. I didn't really like starting off with `is_inivted_user = False`